### ISSUE TITLE
Bugs with uses of display math inline with text

### ISSUE THREAD

**Issue by joelatschool** [RyotaUshio/obsidian-math-booster#181]
It's always been a little buggy, but Obsidian's implementation of markdown allows you to insert display "inline" with text, like so:
```md
A sequence $(u_i)_i$ converges to $v$ if $$
\forall \epsilon > 0 : \exists N \in \mathbb{M} : \forall n \ge N : \left\vert u_n - v  \right\vert < \epsilon.
$$
```

When math booster is enabled, trying to do this in live preview leads to all sorts of bugs, including the equation's code or even the entire paragraph not displaying.

**Comment by RyotaUshio** [comment]
Thank you for reporting.
I think this is essentially the same bug as those reported in #173, and it should have been fixed in the latest beta release [v1.0.3-beta](https://github.com/RyotaUshio/obsidian-math-booster/releases/tag/1.0.3-beta).

Can you test it? You can install this new version using BRAT. (The detailed instruction is in [README](https://github.com/RyotaUshio/obsidian-math-booster#installation))

**Comment by RyotaUshio** [comment]
In my quick test, this problem was not observed with v1.0.3-beta, so I'll close this issue. Feel free to re-open it if you find something is not working well. Thanks!
